.github/workflows/iwyu.yml

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2022 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: include-what-you-use
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'src/**'
+  pull_request:
+    branches:
+      - 'main'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  iwyu:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-22.04
+    timeout-minutes: 180
+    steps:
+      - name: Checkout Orbit
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+          token: '${{ secrets.ORBITPROFILER_BOT_PAT }}'
+          path: orbit
+      - name: Setup git
+        working-directory: ./orbit
+        run: |
+          git config user.name orbitprofiler-bot
+          git config user.email orbitprofiler-bot@google.com
+      - name: Add llvm-15 repo
+        run: |
+          sudo bash -c 'echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list'
+          sudo bash -c 'echo "deb-src http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list'
```

Comment:
Nit: Since you won't need source packages, you will also not need the `deb-src` line.

Reply:
Removed it.